ConcernsCaseWork/ConcernsCaseWork.API/RequestModels/ConcernsRecordRequest.cs

```diff
@@ -6,8 +8,14 @@ public class ConcernsRecordRequest
         public DateTime UpdatedAt { get; set; }
         public DateTime ReviewAt { get; set; }
         public DateTime? ClosedAt { get; set; }
+        
+        [StringLength(300)]
```

Comment:
Sorry if I'm missing something - were there tests for the Concerns Record fields?

Reply:
Ah ignore me - the diff was too big to load in the page :/